CommonTools/RecoAlgos/plugins/ME0MuonTrackCollProducer.cc

```diff
@@ -0,0 +1,73 @@
+
+#include "CommonTools/RecoAlgos/plugins/ME0MuonTrackCollProducer.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/MuonDetId/interface/CSCDetId.h"
+#include "DataFormats/MuonDetId/interface/DTChamberId.h"
+#include "DataFormats/MuonDetId/interface/MuonSubdetId.h"
+//#include "DataFormats/MuonReco/interface/MuonSelectors.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h" 
+#include "RecoMuon/MuonIdentification/plugins/ME0MuonSelector.cc"
+#include <FWCore/Framework/interface/ESHandle.h>
+
+
+// #include "Geometry/GEMGeometry/interface/ME0Geometry.h"
```

Comment:
- here and the rest of the code: please cleanup the commented out code, unless it's really needed for some reason.
- use quotes "" for CMS code includes